src/main/plugins/org.dita.pdf2/xsl/fo/commons.xsl

```diff
@@ -459,6 +459,9 @@ See the accompanying license.txt file for applicable licenses.
 
     <xsl:template name="processTopicNotices">
         <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
+            <xsl:if test="key('map-id', ancestor-or-self::*[contains(@class, ' topic/topic ')][1]/@id)/ancestor::*[contains(@class,' bookmap/backmatter ')]">
```

Comment:
Specifically, I was thinking this - when in back matter use a different attribute set, otherwise no change:

```
<xsl:template name="processTopicNotices">
  <xsl:choose>
    <xsl:when test="back matter">   
      <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.backmatter.notice">
        <xsl:apply-templates select="." mode="processTopicNoticesContents"/>
      </fo:page-sequence>
    </xsl:when>
    <xsl:otherwise>
      <fo:page-sequence master-reference="body-sequence" xsl:use-attribute-sets="page-sequence.notice">
        <xsl:apply-templates select="." mode="processTopicNoticesContents"/>
      </fo:page-sequence>
    </xsl:otherwise>
  </xsl:choose>
</xsl:template>

<xsl:template match="*" mode="processTopicNoticesContents">
  <!-- everything currently in the fo:page-sequence -->
</xsl:template>
```